binding.gyp

```diff
@@ -1,4 +1,8 @@
 {
+  'variables': {
+    'vips_version': '8.9.2',
```

Comment:
Note that this variable must be in-sync with `minimumLibvipsVersion` from `lib/libvips.js`.

https://github.com/lovell/sharp/blob/85459e0ec6d17db55cd4491e64483cfc5e13d843/lib/libvips.js#L13

Perhaps there's a better alternative?

Reply:
The `npm_package_config_libvips` environment variable should contain the value from `package.json` or we could invoke `<!(node -p "require('./package.json').config.libvips")`, which might be a bit more explicit/safer.

Reply:
The problem with `<!(node -p "require('./package.json').config.libvips")` is that it contains the labelled version. https://github.com/lovell/sharp/blob/fbdac804a0104f09a713ce5e59ae0bd95d67f1b6/package.json#L140

We need the unlabelled version (i.e. 8.9.2) for the versioned subdirectory because we can't detect that `-alpha2` at runtime (as far as I know).
https://github.com/lovell/sharp/blob/fbdac804a0104f09a713ce5e59ae0bd95d67f1b6/lib/utility.js#L22-L27


Perhaps we should ensure that the version of `package.json` is always unlabelled?

Reply:
Something like `node -p "require('semver').coerce(require('./package.json').config.libvips).version"` might do the job (and would still not be the most horrible thing in the binding.gyp file).

Reply:
I see. Perhaps we could also just use `minimumLibvipsVersion` from `lib/libvips.js`? Something like this:
```suggestion
    'vips_version': '<!(node -p "require(\'./lib/libvips\').minimumLibvipsVersion")',
```

As an aside,
Is the `console.log` required here?:
https://github.com/lovell/sharp/blob/85459e0ec6d17db55cd4491e64483cfc5e13d843/binding.gyp#L73-L74

I think `node -p` could work here, too.

Reply:
Yes please to both of these, thank you.

Reply:
Fixed with https://github.com/lovell/sharp/pull/2223/commits/3e56eedc42ee06b360f12d7669221001af9a343c.